### DISCUSSION TITLE
Tables in the public schema

### DISCUSSION THREAD

**Discussion by dsebastien** [supabase/supabase#14397] [Questions] [answered]
Up until now, the "public" schema was empty after running `npx supabase init`. This allowed me to then deploy DB migrations from a clean state with Prisma. Unfortunately, there now seems to be multiple tables in the public schema after supabase init:

- extensions
- schema_migrations
- tenants

My questions:
- Is there documentation available anywhere about this?
- Is it intended?
- Will those tables also be present in the production Supabase environment?

**Comment by souvik666** [comment]
yep you can follow this supabase  doc [link](https://supabase.com/docs/guides/integrations/prisma)

**Reply by dsebastien** [reply]
This is not clear for me. I have already configured Prisma and it is functional in production. But now I have a difference between local development and production (those new tables). That documentation page mentions nothing about those tables.

**Comment by dsebastien** [comment]
@sweatybridge, do you have any ideas about this? It is quite troubleshome to have this disparity between the local supabase instance and the one in production. Before this, it was really easy to keep everything in sync since the public schema only contained application-specific tables/elements

**Comment by GaryAustin1** [comment]
Seems like 1.61 may have broken something... Also several users on discord now with the issue.

I believe these tables are supposed to be in _realtime schema based on this:
https://github.com/supabase/realtime/blob/9b331081df96bf424861ac262d316cb1ee3cf190/README.md?plain=1#L67

**Reply by GaryAustin1** [reply, marked as answer]
@dsebastien 
Looks like 1.61.2 when it comes out fixes it.
https://github.com/supabase/cli/pull/1116
